aplib.net-demo/Assets/Scripts/MainMenu/MenuButtons.cs

```diff
@@ -1,48 +1,113 @@
 using UnityEngine;
 using UnityEngine.SceneManagement;
 
-public class MenuPlayButton : MonoBehaviour
+/// <summary>
+/// This class houses all the settings and menu UI buttons/methods.
+/// It is called CanvasManager as it manages all the UI canvi.
+/// </summary>
+public class CanvasManager : MonoBehaviour
 {
     /// <summary>
     /// Reference to the menu canvas.
     /// </summary>
     public GameObject menuCanvas;
```

Comment:
Please keep one empty line between fields and summaries :)
```suggestion
    public GameObject menuCanvas;

```